### ISSUE TITLE
Feature request: hotkey support

### ISSUE THREAD

**Issue by elijahgagne** [pvrs12/Anesidora#6]
Would you be willing to add support for hotkeys? The specific use case I'd like to cover is pausing/playing with hotkeys, hopefully without having the give the plugin focus.

**Comment by pvrs12** [comment]
Sure, I should be able to add that fairly easily. The documentation for it is straightforward (https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/commands) and the additional code should be minimal

I'll update when I've got something working

**Comment by ksariash** [comment]
+1 to this request. Would be nice if this included support for media hotkeys

**Comment by pvrs12** [comment]
I've added support for hotkeys in Firefox. I wasn't able to get anything working for Chrome quite yet. Also, it looks like there is a bug in Firefox currently. All 4 Media keys are being detected identically (as `MediaPlayPause`). As a result I've had to bind the Next Track command to `Alt+N` instead of `MediaTrackNext`. 

TL;DR: 
- Play/Pause: `MediaPlayPause` (or any media key)
- Next Song: `Alt+N`

This should be available in the next version on the store (1.11.0)
